modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/custom/facet/portlet/CustomFacetPortletPreferencesImpl.java

```diff
@@ -35,29 +35,29 @@ public CustomFacetPortletPreferencesImpl(
 	}
 
 	@Override
-	public Optional<String> getAggregationFieldOptional() {
+	public String getAggregationField() {
 		return _portletPreferencesHelper.getString(
 			CustomFacetPortletPreferences.PREFERENCE_KEY_AGGREGATION_FIELD);
 	}
 
 	@Override
-	public String getAggregationFieldString() {
-		Optional<String> optional = getAggregationFieldOptional();
-
-		return optional.orElse(StringPool.BLANK);
+	public String getAggregationField(String defaultValue) {
```

Comment:
@julschong For all the versions of the methods with `defaultValue` parameter in these `*PortletPreferencesImpl`, do you always use it with `StringPool.BLANK` as the default value?

If so, just wrap the non-default version with `GetterUtil.getString()`.

For example,
```
this.getAggregationField(StringPool.BLANK) -> GetterUtil.getString(this.getAggregationField())
```